subcommand/server-acl-init/rules.go

```diff
@@ -11,6 +11,7 @@ type rulesData struct {
 	ConsulSyncDestinationNamespace string
 	EnableSyncK8SNSMirroring       bool
 	SyncK8SNSMirroringPrefix       string
+	GatewayName                    string
```

Comment:
Can you add a comment that this is only set with ingress and terminating gateways.

Reply:
After making the proposed refactor, this no longer applies.